include/PianoRoll.h

```diff
@@ -198,7 +199,8 @@ protected slots:
 		stmaUnmarkAll,
 		stmaMarkCurrentSemiTone,
 		stmaMarkCurrentScale,
-		stmaMarkCurrentChord,
+        stmaMarkCurrentChord,
```

Comment:
Sounds like this is ready to go in once this is fixed.  Just bump the PR so we know it is ready.